modules/administration-guide/partials/proc_collecting-che-metrics-with-prometheus.adoc

```diff
@@ -7,62 +7,127 @@ To use Prometheus to collect, store, and query JVM metrics for {prod-short} Serv
-<4> The scrape target for the metrics from port `8087`. Replace `__<{prod-short}_{orch-namespace}>__` with the {prod-short} {orch-namespace}. The default {prod-short} {orch-namespace} is `{prod-namespace}`.
+<1> The {prod-short} namespace. The default is `{prod-namespace}`.
 ====
 
-. Scale the `Prometheus` Deployment down and up to read the updated ConfigMap from the previous step.
++
+.RoleBinding
+====
+[source,yaml,subs="+quotes,+attributes,+macros"]
+----
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: view-openshift-monitoring-prometheus-k8s
+  namespace: {prod-namespace} <1>
+subjects:
+  - kind: ServiceAccount
+    name: prometheus-k8s
+    namespace: openshift-monitoring
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+----
+<1> The {prod-short} namespace. The default is `{prod-namespace}`.
+====
+
+. Allow the in-cluster Prometheus instance to detect the ServiceMonitor in the {prod-short} namespace.
 +
 [source,terminal,subs="+attributes,quotes"]
 ----
-$ {orch-cli} scale --replicas=0 deployment/prometheus -n monitoring && {orch-cli} scale --replicas=1 deployment/prometheus -n monitoring
+oc label namespace {prod-namespace} openshift.io/cluster-monitoring=true
 ----
 
 .Verification
 
-. Use port forwarding to access the `Prometheus` Service locally:
+. In the *Administrator* view of the OpenShift web console, go to *Observe* -> *Metrics*.
+
+. Run a PromQL query to confirm that the metrics are available. For example, enter `process_uptime_seconds{job="che-host"}` and click *Run queries*.
 +
-[source,terminal,subs="+attributes,quotes"]
+For more information, see link:https://prometheus.io/docs/introduction/first_steps/#using-the-expression-browser[Using the expression browser].
+
+[TIP]
+====
+
+To troubleshoot missing metrics, view the Prometheus container logs for possible RBAC-related errors:
+
+. Get the name of the Prometheus pod:
++
+[source,yaml,subs="+quotes"]
 ----
-$ {orch-cli} port-forward svc/prometheus 9090:9090 -n monitoring
+$ oc get pods -l app.kubernetes.io/name=prometheus -n openshift-monitoring -o=jsonpath='{.items[*].metadata.name}'
 ----
-. Verify that all targets are up by viewing the `targets` endpoint at `localhost:9090/targets`.
-. Use the Prometheus console to view and query metrics:
-** View metrics at `localhost:9090/metrics`.
-** Query metrics from `localhost:9090/graph`.
+
+. Print the last 20 lines of the Prometheus container logs from the Prometheus pod from the previous step:
 +
-For more information, see link:https://prometheus.io/docs/introduction/first_steps/#using-the-expression-browser[Using the expression browser].
+[source,yaml,subs="+quotes"]
+----
+$ oc logs --tail=20 __<prometheus_pod_name>__ -c prometheus -n openshift-monitoring
+----
+
+====
 
+[role="_additional-resources"]
 .Additional resources
 
 * link:https://prometheus.io/docs/prometheus/latest/configuration/configuration/[Configuring Prometheus]
```

Comment:
In case you also need to remove this link like you removed it in `proc_collecting-dev-workspace-operator-metrics-with-prometheus.adoc`.
```suggestion
```

Reply:
Oops, thank you!